### PR TITLE
 #877: Minimize asset root recursive `chown` during installation

### PR DESCRIPTION
**Pull request by tst-ppenev**
* Only recursively `chown` when the asset root directory has incorrect ownership.
* Only `chown` on a Fusion master host.
* Only `chown` once.

### CODE REVIEW

**Comment by tst-nfarah** [comment]
Is this enhancement for 5.2.3 or 5.2.4 ? If it is for 5.2.3 we need to change the target branch from master to 5.2.3.

**Comment by tst-nfarah** [comment]
We need to document this enhancement. If this was meant for 5.2.3,[ release notes for it](https://github.com/google/earthenterprise/blob/release_5.2.3/docs/geedocs/5.2.3/answer/7160003.html) could be an appropriate place.

**Comment by tst-ppenev** [comment]
@tst-nfarah: I'm, actually, not sure if was intended for 5.2.3.  I thought that was already frozen.

File: docs/geedocs/5.2.3/answer/7160003.html [hunk 1]
```diff
@@ -119,8 +119,16 @@ <h5>
...
-            <td>Fixed code so that when you publish DB, GEServer's URL will be used and not the URL which we get from the Publish request message</td>
-          </tr>	  
+            <td>Fixed code so that when you publish DB, GEServer's URL will be used and not the URL which we get from the Publish request message.</td>
+          </tr>        
+          <tr>
+            <td>877</td>
+            <td>Fusion RPM install scripts to only <code>chown</code> the asset root when necessary</td>
+            <td>
+              Recursively <code>chown</code> only when the asset root directory has incorrect ownership on a Fusion master host.
+              This allows upgrading Fusion on machines where changing asset root ownership recursively would be very expensive.
```
Comment (tst-nfarah) [inline docs/geedocs/5.2.3/answer/7160003.html]: minor: `very expensive` --> `time expensive` or `time consuming`

Comment (tst-ppenev) [reply docs/geedocs/5.2.3/answer/7160003.html]: What is the reason for this proposed change?  What is it trying to address?

Comment (tst-nfarah) [reply docs/geedocs/5.2.3/answer/7160003.html]: it was a minor comment: expensive was more a generic term for commodities ...wanted to clarify that we are talking mostly about expensive in term of time (and not $). It is a minor comment that does not hold the PR.

**Pull request merged**